Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+def examples = [
+    'hello-world', 
+    'ping-ipv6',
+]
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '30')),
+    parameters([
+        stringParam(defaultValue: '26', description: 'Set 6LoWPAN Channel', name: 'CHANNEL'),
+        stringParam(defaultValue: '0xabcd', description: 'Set 6LoWPAN PAN ID', name: 'PAN_ID'),
+        stringParam(defaultValue: '1', description: 'Set Serial port to use for console', name: 'PADS'),
+    ])
+])
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/creator-builder:latest"
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
+                userRemoteConfigs: scm.userRemoteConfigs,
+                branches: scm.branches,
+                doGenerateSubmoduleConfigurations: true,
+                submoduleCfg: scm.submoduleCfg,
+                browser: scm.browser,
+                gitTool: scm.gitTool,
+                extensions: scm.extensions + [
+                    [$class: 'CleanCheckout'],
+                    [$class: 'PruneStaleBranch'],
+                    [$class: 'SubmoduleOption', recursiveSubmodules: true]
+                ],
+            ])
+        }
+        stage('Build') {
+            for (item in examples) {
+                sh "cd examples/${item} && make TARGET=mikro-e USE_CA8210=1 CHANNEL=${params.CHANNEL} \
+                PAN_ID=${params.PAN_ID} USE_SERIAL_PADS=${params.PADS}"
+                sh "cd examples/${item} && xc32-bin2hex *.mikro-e && mv *.hex ${WORKSPACE}/"
```

Comment:
Why bother moving them if you can just archive them here? Then you get the added benefit that even if one example fails all previous successful ones would still get uploaded. Also in previous versions of this review you were specifying exact file names not using *. This change could cause issues